examples/glue/glue_entrypoint.ipynb

```diff
@@ -0,0 +1,370 @@
+    "\n",
+    "In all logging instances, such as Weights and Biases and in the results table outputted at the end of training, all the fine-tune runs will be grouped by pre-train checkpoint name for easier organization and run tracking.\n",
+    "\n",
+    "Below is an example `finetune_hparams` that loads checkpoints from an Amazon S3 bucket:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    finetune_hparams:\n",
+    "      ...\n",
+    "      finetune_ckpts:\n",
+    "        - path/to/checkpoint1\n",
+    "        - path/to/checkpoint2\n",
+    "\n",
+    "      # if paths are in ObjectStore, the below is expected to be defined\n",
+    "      loggers:\n",
+    "        object_store:\n",
+    "          object_store_hparams: &bucket                 # Aliases the bucket information under bucket\n",
+    "            s3:\n",
+    "              bucket: your-bucket-here\n",
+    "      load_object_store:\n",
+    "        *bucket                                         # Refers to the previously defined bucket alias"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "❗ Note: The load paths provided in `finetune_ckpts` have to be relative paths within an object store bucket/local directory as Composer does not currently allow checkpoints to be loaded via remote URIs.  Alternatively, you can provide a full https URL to a remote checkpoint as your full path, such as `https://storage.googleapis.com/path/to/checkpoint.pt`, and you will no longer need to define the `load_object_store` parameter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pre-training and fine-tuning\n",
+    "\n",
+    "To run the entire end-to-end pipeline, you are expected to provide the entrypoint with your pre-train configuration as explained above, as well as any overrides to apply to the fine-tuning jobs. In this case, the entrypoint is run in two distinct stages for distributed pre-training and multiprocessed fine-tuning; however, all information transferred between the stages is automatically handled by the entrypoint.  Checkpoints are automatically saved to your specified `save_folder` and loaded from wherever pre-training saved them, therefore the `finetune_ckpts` section of `finetune_hparams` is ignored if specified."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "❗ Note: The entrypoint runs all 8 GLUE fine-tuning tasks on every saved pre-training checkpoint, so set your [save_interval](https://docs.mosaicml.com/en/stable/trainer/checkpointing.html?highlight=sve%20interval#save-interval) within your `pretrain_hparams` appropriately to avoid unnecessarily long evaluation times. "
```

Comment:
```suggestion
    "❗ Note: The entry point runs all 8 GLUE fine-tuning tasks on every saved pre-training checkpoint, so set your [save_interval](https://docs.mosaicml.com/en/stable/trainer/checkpointing.html?highlight=sve%20interval#save-interval) within your `pretrain_hparams` appropriately to avoid unnecessarily long evaluation times. "
```